internal/impl/sql/integration_test.go

```diff
@@ -718,6 +758,78 @@ func TestIntegrationPostgres(t *testing.T) {
 	testSuite(t, "postgres", dsn, createTable)
 }
 
+func TestIntegrationSpanner(t *testing.T) {
+	integration.CheckSkip(t)
+
+	pool, err := dockertest.NewPool("")
+	if err != nil {
+		t.Skipf("Could not connect to docker: %s", err)
+	}
+	pool.MaxWait = 10 * time.Second
```

Comment:
```suggestion
	pool.MaxWait = 3 * time.Minute
```

just so it's the same as the other tests in this file